CentrED/Map/MapManager.cs

```diff
@@ -742,6 +752,17 @@ public void Update(GameTime gameTime, bool isActive, bool processMouse, bool pro
             _prevKeyState = keyState;
         }
 
+        // Check if we're drawing with the mouse button
+        bool isDrawing = isActive && Mouse.GetState().LeftButton == ButtonState.Pressed;
+        
+        // If we were drawing but stopped, reset the sequence
+        if (_wasDrawing && !isDrawing && UseSequentialTileSet)
+        {
+            ResetSequence();
```

Comment:
I believe ResetSequence() could be just put at the end of BaseTool.OnMouseReleased()
Having it in MapManager feels like a little bit too much.